code/game/objects/items/weapons/gift_wrappaper.dm

```diff
@@ -316,7 +316,8 @@
 
 	var/atom/movable/I = new gift_type(M)
 	M.remove_from_mob(src)
-	M.put_in_hands(I)
+	if (!M.put_in_hands(I))
```

Comment:
gifts were getting an error trying to call dropped on mobs, after failing to put them in hands.